src/datetime.js

```diff
@@ -589,9 +590,20 @@ export default class DateTime {
       return DateTime.invalid(unsupportedZone(zoneToUse));
     }
 
+    let normalized;
+    try {
+      normalized = normalizeObject(obj, normalizeUnit, [
+        "zone",
+        "locale",
+        "outputCalendar",
+        "numberingSystem"
+      ]);
+    } catch (error) {
+      return DateTime.invalid(error.message);
```

Comment:
I'm not sure about this. The rule in Luxon is that we want to throw on programmer errors and invalidate on data errors. That roughly translates to bad keys and bad types should throw, out-of-range numbers or unknown strings should invalidate. So I think here we should probably just throw all the way up.

Reply:
Makes sense. I thought the idea was to avoid throwing and use invalid with a reason instead.

But that would be inconsistent with Duration.